manifests-maistra/1.0.1/maistraoperator.v1.0.1.clusterserviceversion.yaml

```diff
@@ -114,6 +114,7 @@ spec:
     url: https://github.com/Maistra/istio-operator
   - name: Istio
     url: https://istio.io/
+  replaces servicemeshoperator.v1.0.0
```

Comment:
should this be `maistraoperator.v1.0.0`?

Reply:
It should be, fixed.